CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing
+
+## Set up
+
+After cloning this repo, create a [virtualenv](https://virtualenv.pypa.io/en/stable/) and ensure dependencies are installed by running:
```

Comment:
Could we offer [pipenv](https://docs.pipenv.org/en/latest/) as a lighter weight alternative?

Reply:
@mvanlonden from my personal experience the packaging story of Python is in flux at the moment, I'd rather pick the basic one and when things settle down and a clear winner appears we move over to use it.

Reply:
Agree on Pipenv. For personal projects, it works fine and has a lot of nice convenience features but there are some things that make it less desirable for cross platform development and especially building libraries.

Reply:
"lighter weight" ?  what's wrong with `virtualenv` ?